src/harness/harness.ts

```diff
@@ -732,7 +732,8 @@ module Harness {
     }
 
     // Settings 
-    export var userSpecifiedroot = "";
+    export let userSpecifiedRoot = "";
+    export let lightMode = false;

```

Comment:
Since light mode only skips invariant checking now, I don't think this applies anymore.